deltacat/utils/arguments.py

```diff
@@ -0,0 +1,26 @@
+import inspect
+import copy
+from typing import Any, Dict
+
+
+def sanitize_kwargs_to_callable(callable: Any, kwargs: Dict) -> Dict:
```

Comment:
Is this strictly needed? Do we have any mechanism to at least track original arguments and sanitized arguments?

Reply:
This is most generic I could get. Do you have any ideas?

Reply:
I think we should at least log pre/post sanitization inputs somewhere to expose dropped kwargs.